code/datums/miscreant_objectives.dm

```diff
@@ -119,4 +122,62 @@ ABSTRACT_TYPE(/datum/objective/miscreant)
 	spy
 		explanation_text = "Become an inter-galactic spy and refer to everyday objects as 'gadgets'."
 
+ABSTRACT_TYPE(/datum/objective/miscreantrp)
+/datum/objective/miscreantrp
+	shoes
+		explanation_text = "Through trade, sweet-talking, light extortion, blackmail, and other devilish, but non-violent means, secure and deep-fry as many shoes as possible"
+
+	bar
+		explanation_text = "Set up and aggressively market an illicit bar to compete with the Barman, try and draw as many customers away as possible!"
+
+	kitchen
+		explanation_text = "Set up and aggressively market an illicit kitchen to compete with the Chef, try and draw as many customers away as possible!"
+
+	salvage
+		explanation_text = "Gather as many items as possible from maint-tunnels and spare rooms, try and sell them to the crew for a profit."
+
+	secret
+		explanation_text = "Spread misinformation about a terrible event to incite a panic among the crew. If caught, claim it was a safety drill."
+
+	ai
+		explanation_text = "Start and deliver a petition to the captain to secure equal rights for the AI and cyborgs. Try and get as many signatures as you can."
+
+	union
+		explanation_text = "Work with your coworkers to found a union. Once formed, make increasingly radical demands for pay and benefits to command."
+
+	museum
+		explanation_text = "Found and curate a museum."
+
+	party
+		explanation_text = "Single out a crew member and throw them a surprise party."
+
+	mascot
+		explanation_text = "Go on a campaign to establish a station mascot."
+
+	shrubs
+		explanation_text = "Destroy as many shrubs as you can. Replace the destroyed shrubs with other objects. If caught, claim to be balancing the Feng-Shui of the station."
+
+	detective
+		explanation_text = "Found a private detective agency and attempt to solve cases before the detective can. Come up with extremely absurd explanations for crimes and insist that security is secretly in on it."
+
+	exterminator
+		explanation_text = "Kill as many non-monkey, non-pet animals aboard the station as possible and bring their corpses to the bridge. Once finished, claim to be a trained exterminator and demand payment for your services."
+
+	conspiracy
+		explanation_text = "Spin wild theories about how every single accident or problem aboard the station is the fault of some shadowy NT/syndicate cabal. Attempt to get on the radio show and share your theories with the world."
```

Comment:
this has the potential to get really annoying and repetitive. people do this gimmick often enough without being *asked* to do it.

Reply:
removed and replaced with a safer objective from the regular miscreant pool.